docs/content/gs-adding-users-proc.adoc

```diff
@@ -35,39 +28,69 @@ By default, when you log into Red Hat Single Sign-On administration console, the
+      "firstName": "John",
+      "lastName": "Doe",
+      "realmRoles": ["offline_access", "uma_authorization"],
+      "clientRoles": {
+          "account": ["manage-account", "view-profile"]
+      },
+      "groups": [],
+      "attributes": {},
+      "credentials": [{
+        "type" : "password",
+        "value" : "Password1"
+      }]
+    },
+    {
+      "username": "msmith",
+      "enabled": true,
+      "email": "marysmith@example.com",
+      "firstName": "Mary",
+      "lastName": "Smith",
+      "realmRoles": ["offline_access", "uma_authorization"],
+      "clientRoles": {
+          "account": ["manage-account", "view-profile"]
+      },
+      "groups": [],
+      "attributes": {},
+      "credentials": [{
+        "type" : "password",
+        "value" : "Password1"
+      }]
+    }
+  ]
+}
+----
 
 === Setting User Password
 - Navigate to the *Users* menu.
 - Find the user you wish to edit and click on their *Edit* button associated with the user.
 - Go to the *Credentials* tab.
 - Set a password for the user and click the *Reset Password* button to save.
-- Set the `Temporary` field to *ON* only if you require the user to update their password upon login. 
-
-image:../assets/images/gs-adding-users-sso-set-user-password.png[Setting a password for a user in SSO]
-
-The newly created user should now be ready to be used to login to the OpenShift console and all the Integreatly services.
+- If you require the user to update their password upon login, set the `Temporary` field to *ON*.
 
 == Deleting Users in the SSO Admin Console
 - Navigate to the *Users* menu
 - Click the *View all users* button at the top of the page.
 - Find the user you wish to delete
 - Click the *Delete* button associated to the user you wish to remove.
-
-image:../assets/images/gs-adding-users-sso-delete-user.png[Delete a user in SSO]
-
 - Click *Delete* in the confirmation box to proceed.
 
-image:../assets/images/gs-adding-users-sso-user-delete-confirmation.png[Delete user confirmation]
-
-The user should now be deleted from the SSO and will no longer be available to login to the OpenShift console and Integreatly services.
-
 == User Management in 3Scale
-Users created in the OpenShift SSO can be used to login to the 3Scale console. The user's account will only be created in 3Scale once a user attempts to login to the 3Scale console. The user created is assigned the role `member` and no permissions set by default. 
+Users created in  Red Hat Single Sign-On can login to the 3Scale console, however the user's account is created in 3Scale after that initial login. The user created is assigned the role `member` and no permissions set by default. 
 
 The user role and permission can only be set once a user account is created in 3Scale. Because of this, the only way to change the user's role/permission before the user attempts to login to the 3Scale console is to create a user account that matches the account created in the OpenShift SSO.
```

Comment:
This bit will need to be changed/removed to accommodate the new user workflow. 

```Because of this, the only way to change the user's role/permission before the user attempts to login to the 3Scale console is to create a user account that matches the account created in the OpenShift SSO```

Instead of telling them to create a user, it should state that the user may request for and will need to be given permissions after they have logged in (when that user account is created).